sdk/ai/azure-ai-projects/azure/ai/projects/_version.py

```diff
@@ -6,4 +6,4 @@
 # Changes may cause incorrect behavior and will be lost if the code is regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "1.0.0b2"
+VERSION = "1.0.0b3"
```

Comment:
This has already been done in the feature branch. Please rebase before sending PR.

Reply:
Rebased